src/main/java/ca/corefacility/bioinformatics/irida/web/controller/api/projects/RESTProjectAnalysisController.java

```diff
@@ -65,6 +66,7 @@ public RESTProjectAnalysisController(ProjectService projectService,
 	 *         {@link Project}.
 	 */
 	@RequestMapping(value = "/api/projects/{projectId}/analyses", method = RequestMethod.GET)
+	@ResponseBody
```

Comment:
Since the Azure sdk required jackson >2.10 the tests for this as well as the AnalysisAjaxTableController tests broke. My guess is the response is not being serialized properly

Reply:
Ok.  So I think something is going funny that made this necessary.  Our REST config in https://github.com/phac-nml/irida/blob/development/src/main/java/ca/corefacility/bioinformatics/irida/config/web/IridaRestApiWebConfig.java should serialize any resource to JSON automatically without the `@ResponseBody` annotation.  When I'm going through the rest api with curl I'm noticing some controllers being pretty-printed and some that aren't.  Pretty print is set up in https://github.com/phac-nml/irida/blob/development/src/main/java/ca/corefacility/bioinformatics/irida/config/web/IridaRestApiWebConfig.java#L73 and I think should be universal through the REST API.  

So long story short, something funny is going on to need this.  I think we'll have to dig because there's something going wrong with configuration somehow.

Reply:
Sure I will take a look into it and see if I can figure out what's going on

Reply:
So now the tests are passing..totally weird. Updated in 23c0b6d48cdf222b4c5674c280f79ed361766082

Reply:
Updated with PR #769